common/wonders/wc_wonders_buildable.txt

```diff
@@ -364,3 +364,118 @@ wonder_soul_engine = {
 		}
 	}
 }
+wonder_statue_yak = {
+	picture = GFX_wonder_strip_yak_statue
+	allow_coastal = yes
+	potential = {
+		# Warcraft
+		evil_public_religion_trigger = no
+		
+		culture = yaungol
```

Comment:
Why the grummles who also worship yaks can't build it?

Reply:
it can be done but some wonder upgrades are more yaungol focused so it would be kinda awkward.

Reply:
> it can be done but some wonder upgrades are more yaungol focused so it would be kinda awkward.

Don't see a problem here. Like you build yak statue to improve relations with yaungol neighbours.